### ISSUE TITLE
Problem when loading data into graph

### ISSUE THREAD

**Issue by thokari** [angularjs-nvd3-directives/angularjs-nvd3-directives#180]
Hi, 
I think I am missing something very simple here.
I want to load several datasets in parallel and show them in one graph.
Unfortunately, when I initalize the graphData variable with an empty array, the picture just says, 'No Data available', regardless if I put something into it later on.

I solved the problem, by not creating the actualGraphData until some timeout, but this feels like a hack. I saw another Issue, but couldn't figure out what the heck DIV the guy was talking about...

Heres the code:

```
function GraphController($scope, Events, Evaluate, $timeout) {

$scope.graphData = [];

$scope.eventType = 'test_1';
$scope.fromSpec = { 'from.minutes': 60 };
$scope.accumulators = ['avg'];
$scope.resolution = 'm5';

function makeGraphDataEntry(name, values) {
    return {
        key: name, 
        values: values
    };
}

$timeout(function() {
    $scope.actualGraphData = $scope.graphData;
}, 3000)

Events.get($scope.eventType, $scope.fromSpec).success(function(data) {
    $scope.graphData.push(makeGraphDataEntry($scope.eventType, data));
});

angular.forEach($scope.accumulators, function(acc) {
    Evaluate.get($scope.eventType, acc, $scope.resolution, $scope.fromSpec).success(function(data) {
        $scope.graphData.push(makeGraphDataEntry(acc, data));
    });
})

$scope.xAxisFormat = function() {

    function pad(num, size) {
        var s = num + "";
        while (s.length < size)
            s = "0" + s;
        return s;
    }

    return function(entry) {
        var date = new Date(entry);
        var hours = date.getHours();
        var mins = date.getMinutes();
        var secs = date.getSeconds();
        return pad(hours, 2) + ":" + pad(mins, 2) + ":" + pad(secs, 2);
    }
};
```

}

The template looks like this:
<code>
     &lt;nvd3-line-chart data="actualGraphData" ng-model="actuaGraphData" height="400"
                    showXAxis="true" showYAxis="true" xAxisTickFormat="xAxisFormat()"
                    tooltips="true" useInteractiveGuideLine="true" showLegend="true">
                &lt;/nvd3-line-chart>
</code>
